### PR TITLE
Update library for compatibility with latest Slate

### PR DESCRIPTION
**Pull request by ahorner**
- Import latest `slate` assets
- Add configuration variables for logo image and search functionality
- Rewrite `$.tocify` for compatibility with new `slate` version
- Update layout for parity with new `slate` styling
- Allow for per-request response attributes
- Remove extraneous rake tasks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.DS_Store
```
Comment (BenMorganIO) [inline .gitignore:1]: Note: you may also want to add this to your global .gitignore

File: lib/tasks/apitome_slate_tasks.rake [hunk 2]
```diff
@@ -1,13 +0,0 @@
...
-RSpec::Core::RakeTask.new('docs:generate') do |t|
-  t.pattern = 'spec/acceptance/**/*_spec.rb'
-  t.rspec_opts = ["--format RspecApiDocumentation::ApiFormatter"]
-end
-
-desc 'Generate API request documentation from API specs (ordered)'
-RSpec::Core::RakeTask.new('docs:generate:ordered') do |t|
-  t.pattern = 'spec/acceptance/**/*_spec.rb'
-  t.rspec_opts = ["--format RspecApiDocumentation::ApiFormatter", "--order defined"]
-end
```
Comment (BenMorganIO) [inline lib/tasks/apitome_slate_tasks.rake:13]: Why do I feel like this was important at some point for some reason?

**Comment by BenMorganIO** [comment]
@ahorner thank you for updating this project, you're amazing! Changes look good, would you like me to merge this?

Also, will you be actively using this repository? If so, I can make you a collaborator if you'd like to merge and release your own versions.

**Comment by ahorner** [comment]
Feel free to merge away!

The only thing I'm anticipating adding beyond this PR is support for including Apitome's `formatted_readme` in the final output, but there may be bits and bobs I notice as I'm working through my use case. Collaborator access would be wonderful, but that's totally up to you.

**Pull request merged**